.github/workflows/ci.yml

```diff
@@ -15,12 +15,16 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
-      - name: Set up Python 3.9
-        uses: actions/setup-python@v3
+      - uses: actions/setup-python@v4
         with:
           python-version: "3.9"
-      - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip
-          pip install pytest
+          cache: "pip" # caching pip dependencies
+          cache-dependency-path: "requirements-dev.txt"
+      - run: pip install -r requirements-dev.txt
+      - name: Cache pre-commit
+        uses: actions/cache@v3
+        with:
+          path: ~/.cache/pre-commit
+          key: pre-commit-3|${{ env.pythonLocation }}|${{ hashFiles('.pre-commit-config.yaml') }}
```

Comment:
Is this cache used?

Reply:
It's what the pre-commit action does, so presumably pre-commit uses it.

Reply:
Not anymore, since I removed pre-commit again. Good catch.